tests/data/EventFactory.ts

```diff
@@ -41,8 +41,8 @@ export class EventFactory {
       deleted: false,
       eventLink: faker.internet.url(),
       thumbnail: FactoryUtils.getRandomImageUrl(),
-      discordEvent: uuid(),
-      googleCalendarEvent: uuid(),
+      discordEvent: faker.datatype.hexaDecimal(10),
```

Comment:
Is this the way discord generate Uuid or is this just random string?

Reply:
This is just a random string. Discord UUIDs are different (see this error message: https://discord.com/channels/573028991527550986/1012204566122598431/1236042800043397261) than what Postgres allows for UUIDs (https://www.postgresql.org/docs/current/datatype-uuid.html), which is why I opted to make it a string